Library/Homebrew/cmd/cleanup.rb

```diff
@@ -64,12 +64,13 @@ def cleanup_formula f
   end
 
   def cleanup_keg keg
+    size = `/usr/bin/du -sh #{keg} | /usr/bin/awk -F'[^0-9]*' '{print $2}'`.strip
```

Comment:
Would be nice to pull this logic into Ruby rather than using pipes. Maybe worth looking at how `info.rb` does it?